okoroemeka/Lender_Frontend
# ft(admin dashboard): add loan details

#### What does this PR do?
Add loan details to modal

#### Description of Task to be completed?
Add loan details to modal when a loan is clicked on to be viewed by an admin

#### How should this be manually tested?
- Clone the repo run `yarn install`, then check out to the branch
- signin as an admin, then click on any to view the loan details or react to it

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[Finishes #169799506](https://www.pivotaltracker.com/story/show/169799506)
